lib/phoenix/controller.ex

```diff
@@ -618,6 +618,8 @@ defmodule Phoenix.Controller do
   end
 
   defp do_render(conn, template, format, assigns) do
+    import Phoenix.Endpoint, only: [instrument: 3, instrument: 4]
```

Comment:
Why are we importing both functions? Is it even worth importing if it is a single call?